internal/authclient/authclient.go

```diff
@@ -120,6 +120,7 @@ func (client *AuthClient) runOpenBrowser(ctx context.Context, li net.Listener, s
 	}
 
 	transport := &http.Transport{
+		Proxy:           http.ProxyFromEnvironment,
```

Comment:
This makes sense to me. I wonder if a broader fix would be to use a Clone'd version of the default http transport `http.DefaultTransport` which also has some likely sane default timeouts and settings beyond Proxy?

https://golang.org/pkg/net/http/#Transport.Clone

Reply:
Yeah that sounds good to me. You mean something like this right?

```
transport := http.DefaultTransport.Clone()
transport.TLSClientConfig = client.cfg.tlsConfig
```

Reply:
Yep!